tests/components/homeassistant_sky_connect/test_config_flow.py

```diff
@@ -246,7 +242,7 @@ def mock_detect_radio_type(radio_type=RadioType.ezsp, ret=True):
     async def detect(self):
         self.radio_type = radio_type
         self.device_settings = radio_type.controller.SCHEMA_DEVICE(
-            {CONF_DEVICE_PATH: self.device_path}
+            {"path": self.device_path}
```

Comment:
Instead of using the string here, we could also use the following import `from zigpy.config import CONF_DEVICE_PATH`